lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue

```diff
@@ -0,0 +1,190 @@
+				</div>
+			</div>
+			<icon-button
+				v-if="isClosable"
+				:icon="ICONS.FA_XMARK"
+				:size="ICON_BUTTON_SIZES.MEDIUM"
+				@click="$emit('close')"
+			/>
+		</div>
+		<divider v-if="hasDivider" :size="DIVIDER_SIZES.L" />
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/colors/tokens';
+
+.drawerHeader {
+	display: flex;
+	flex-direction: column;
+
+	&__secondLevel {
+		position: absolute !important; //it is required so firstLevel content does not make component wider when hidden, and important is needed so component does not change its width when button was clicked
+	}
+
+	&__leftIcon {
+		color: $color-neutral-icon;
+	}
+
+	&__firstLevel {
+		display: flex;
+		flex-direction: column;
+		min-width: 0;
+		row-gap: $space-xxxxs;
+
+		&.-hidden {
+			visibility: hidden; //by this we make sure that height does not change when switching to second level
+		}
+	}
+
+	&__eyebrow {
+		@include info-s-extensive-bold-uppercase;
+
+		color: $color-neutral-text-weak;
+
+		&.-isInteractive:hover {
+			color: $color-neutral-text;
+			cursor: pointer;
+		}
+
+		&.-ellipsis {
+			overflow: hidden;
+			text-overflow: ellipsis;
+			white-space: nowrap;
+		}
+	}
+
+	&__title {
+		@include heading-s-default-bold-uppercase;
+
+		align-items: center;
+		color: $color-neutral-text-strong;
+		column-gap: $space-xxxs;
+		display: flex;
+
+		.-ellipsis {
+			overflow: hidden;
+			text-overflow: ellipsis;
+			white-space: nowrap;
+		}
+	}
+
+	&__titleWrapper {
+		align-items: center;
+		column-gap: $space-xxxxs;
+		display: flex;
+		justify-content: space-between;
+		min-height: 82px;
+		padding: $space-m $space-xs $space-m $space-s;
+	}
+}
+</style>
+
+<script>
+import Button from '../../Buttons/Button/Button.vue';
+import Divider from '../../Divider/Divider.vue';
+import IconButton from '../../Buttons/IconButton/IconButton.vue';
+import Pill from '../../Pill/Pill.vue';
+import Icon from '../../Icons/Icon/Icon.vue';
+import { BUTTON_TYPES } from '../../Buttons/Button';
+import { ICONS, ICON_SIZES } from '../../Icons/Icon';
+import { DIVIDER_SIZES } from '../../Divider';
+import { ICON_BUTTON_SIZES } from '../../Buttons/IconButton';
+
+export default {
+	name: 'DrawerHeader',
+	components: {
+		DsButton: Button,
+		Divider,
+		Icon,
+		IconButton,
+		Pill,
+	},
+	props: {
+		title: {
+			type: String,
+			required: true,
+		},
+		eyebrowText: {
+			type: String,
+			default: null,
+		},
+		pillLabel: {
+			type: String,
+			default: null,
+		},
+		leftIcon: {
+			type: [Object, null],
+			default: null,
+		},
+		isClosable: {
+			type: Boolean,
+			default: true,
+		},
+		isSecondLevel: {
+			type: Boolean,
+			default: false,
+		},
+		isInteractiveEyebrow: {
+			type: Boolean,
+			default: false,
+		},
+		hasDivider: {
+			type: Boolean,
+			default: false,
+		},
+		eyebrowEllipsis: {
```

Comment:
i don't se this props in [Miro](https://miro.com/app/board/o9J_knPnexA=/?moveToWidget=3458764551320412081&cot=14), it is new? should we update it?

Reply:
https://www.figma.com/file/oqNhXXGdc4ZnNQ6YNpkLEK?node-id=1823:641360#431804422